changelogs/fragments/hostmacro_modules.yml

```diff
@@ -0,0 +1,2 @@
+minor_change:
+  - zabbix_hostmacro module - Add description property for Host macro creation/update: allow to set/update description of Zabbix host macros.
```

Comment:
This file failed during sanity checks, I think because of the colon you have in here.

Reply:
yes you can't use colons here.